src/library/scala/collection/immutable/Set.scala

```diff
@@ -201,6 +244,20 @@ object Set extends ImmutableSetFactory[Set] {
     override def forall(@deprecatedName('f) p: A => Boolean): Boolean = {
       p(elem1) && p(elem2) && p(elem3)
     }
+    override private[scala] def filterImpl(pred: A => Boolean, isFlipped: Boolean): Set[A] = {
```

Comment:
This patch comes from https://github.com/scala/scala/commit/20b4596bc340ccfc5ba37847d6b21215586f0398#diff-e4f24e9fd33bb1494a43fee25d3b712fR260